xtask/src/util.rs

```diff
@@ -20,7 +20,7 @@ pub fn get_cargo_workspace() -> &'static Path {
         cross::cargo_metadata_with_args(
             Some(manifest_dir.as_ref()),
             None,
-            &mut MessageInfo::create(true, false, None).expect("should not fail"),
+            &mut MessageInfo::create(2, false, None).expect("should not fail"),
```

Comment:
Once again, this can just use the normal `Verbose(1)` level, since the only difference will be if we print the command. We also ignore the level in `run_and_get_output`, etc.